bin/update_ati_analytics.py

```diff
@@ -0,0 +1,125 @@
+"""
```

Comment:
```suggestion
#!/usr/bin/env python3
"""
```
And a `chmod +x` would let us run this script directly

Reply:
executable mode added to the file and py3 bin header